bin.src/extractPhosimGainSaturation.py

```diff
@@ -39,12 +39,13 @@
 def read_files(path, verbose=False):
     """Return a dictionary of amp name: (gain, saturation) for all amp files in path."""
     amps = {}
-    files = glob.glob(os.path.join(path, 'lsst_*_R??_S??_C??*.fits.gz'))
+    files = glob.glob(os.path.join(path, 'lsst_a_*R??_S??_C??*.fits.gz'))
     for file in files:
         if verbose:
             print(file)
         header = fits.getheader(file)
         ampid = '_'.join((header['CCDID'], header['AMPID']))
+#        ampid = header['CHIPID']
```

Comment:
Let's delete this commented out line since I think it was added to address the issue that is actually fixed by making sure amplifier files are used in this test in the globbing at line 42.